src/screens/WhotoFollow/elements/ItemUser.js

```diff
@@ -49,6 +50,15 @@ const ItemUser = ({photo, username, bio, followed, onPress, userid}) => {
 };
 
 // const MemoItemUser = React.memo(ItemUser, isPropsEqual);
+ItemUser.propTypes = {
+  photo: PropTypes.string,
+  username: PropTypes.string,
+  bio: PropTypes.string,
+  followed: PropTypes.array,
+  onPress: PropTypes.func,
+  userid: PropTypes.string,
+  karmaScore: PropTypes.number
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `karmaScore` prop is added to the `ItemUser` component and its type is correctly defined as a number in PropTypes. However, it would be good to add a default value for `karmaScore` in case it's not provided. This will prevent potential issues with rendering the `CircleGradient` component if `karmaScore` is undefined.
```diff
  ItemUser.propTypes = {
    photo: PropTypes.string,
    username: PropTypes.string,
    bio: PropTypes.string,
    followed: PropTypes.array,
    onPress: PropTypes.func,
    userid: PropTypes.string,
+   karmaScore: PropTypes.number.isRequired,
  };

+ ItemUser.defaultProps = {
+   karmaScore: 0,
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->